### PR TITLE
doc(zeebe): add standalone decision evaluation support

### LINKED ISSUE #1516: I can read about standalone Decision evaluation
Zeebe will support standalone Decision evaluation (see https://github.com/camunda/zeebe/issues/11037). We will need to document this feature.

**Tasks:**
- [x] Document the call in the [Zeebe API (gRPC)](https://docs.camunda.io/docs/apis-clients/grpc/)
    - Document the different responses:
       - [x] Invalid command - rejection
       - [x] Evaluation failure - valid response with failure message
       - [x] Evaluation success - valid response with result
    - Document that if both the Decision ID and KEY are provided, the ID takes presedence.
- [x] Add an example for the Java client ([similar to this maybe](https://docs.camunda.io/docs/apis-clients/java-client-examples/process-instance-create/))
- [x] Mention the new API in [the Business rule task docs](https://docs.camunda.io/docs/components/modeler/bpmn/business-rule-tasks/)

**Blocked by**
* https://github.com/camunda/zeebe/issues/11047
* https://github.com/camunda/zeebe/issues/11048

### PR DESCRIPTION
**Pull request by koevskinikola**
## What is the purpose of the change

Document support for standalone decision evaluation.

closes #1516 

## Are there related marketing activities

/

## When should this change go live?

The change should go live with the release of Zeebe `8.2.0`.

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

## Blocked by
- [x] Zeebe `8.2.0-alpha4` release
- [x] https://github.com/camunda-community-hub/camunda-8-examples/pull/91

### CODE REVIEW

**Comment by koevskinikola** [comment]
For the purposes of the Java client example, I created a PR in the [camunda-8-examples repo](https://github.com/camunda-community-hub/camunda-8-examples). The code and files referenced by the Java examples can be found [here](https://github.com/camunda-community-hub/camunda-8-examples/pull/87).

File: docs/apis-clients/grpc.md [hunk 1]
```diff
@@ -286,6 +286,128 @@ Returned if:
...
+### `EvaluateDecision` RPC
+
+Evaluates a decision. You specify the decision to evaluate either by
+using its unique KEY (as returned by DeployResource), or using the decision
+ID. When using the decision ID, the latest deployed version of the decision
+is used.
+
+:::note
+When you specify both the decision ID and KEY, the ID is used to find the decision to be evaluated.
+:::
```
Comment (remcowesterhoud) [inline docs/apis-clients/grpc.md:298]: 👍 Thanks for adding this

**Review by remcowesterhoud — APPROVED** [review]
From a Zeebe point of view it looks good to me 👍

**Comment by pepopowitz** [comment]
FYI I've merged `main` into this branch, to take advantage of #1658, and get `build-docs` (hopefully) succeeding again.

**Review by christinaausley — APPROVED** [review]
Nice. Very minor change in my latest commit for the tabbed list.

**Comment by koevskinikola** [comment]
After a [discussion with Bernd](https://github.com/camunda-community-hub/camunda-8-examples/pull/87#issuecomment-1411699399) in the [camunda-8-examples PR](https://github.com/camunda-community-hub/camunda-8-examples/pull/87), let's block the merge of this PR until Zeebe `8.2.0-alpha4` is released.

Reason: For the Java client example, we need to provide some code in the `camuna-8-examples` repo. To do this, we need to bump the Zeebe version in `camuna-8-examples`, and we should't use Zeebe `SNAPSHOT` versions in the examples.

**Pull request merged**